sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 * Session receivers are now created via their own top level functions, e.g. `get_queue_sesison_receiver` and `get_subscription_session_receiver`.  Non session receivers no longer take session_id as a paramter.
 * `ServiceBusSender.send()` no longer takes a timeout parameter, as it should be redundant with retry options provided when creating the client.
 * Exception imports have been removed from module `azure.servicebus`. Import from `azure.servicebus.exceptions` instead.
+* `ServiceBusSender.schedule()` has swapped the ordering of parameters `schedule_time_utc` and `messages`.
```

Comment:
```suggestion
* `ServiceBusSender.schedule()` has swapped the ordering of parameters `schedule_time_utc` and `messages` for better consistency with `send()` syntax.
```

(I figure giving at least the bit of a reason when possible makes the blow slightly softer?)